### DISCUSSION TITLE
Clone user.js from Main Browser to all (or selected profiles)

### DISCUSSION THREAD

**Discussion by sapphired18** [filips123/PWAsForFirefox#220] [Ideas]
i have some configuration and made it into user.js in main browser, can FirefoxPWA do automatically to copy to all PWA profiles?

**Comment by filips123** [comment]
No, there is no way of copying configuration to PWA profiles automatically. For new profiles, I plan to create support for some profile templates which can be copied to new profiles if enabled (follow #219 for that). For existing profiles, you would probably have to do this manually, but it can probably be automated with some simple scripts.
